src/Classes/ConfigTab.lua

```diff
@@ -422,6 +448,10 @@ local ConfigTabClass = newClass("ConfigTab", "UndoHandler", "ControlHost", "Cont
 					return false
 				end))
 			end
+			if isShowAllConfig(varData) then
```

Comment:
you can just call this directly when calling the shownFunc() no? Instead of having this override variable.

E.g move the self.toggleConfigs check to the isShowAllConfig function and then simply do 

`if not shownFunc() and not isShowAllConfig(varData)`